src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -258,16 +258,9 @@
       <string/>
      </property>
      <property name="title">
-      <string extracomment="Sets default action when double-clicking a track in the library.">Track Load Action</string>
+      <string extracomment="Sets default action when double-clicking a track in the library.">Track Double Click Action</string>
```

Comment:
This breaks string freeze

Reply:
Yes, it does unfortunately. The old text is not really accurate anymore. If you insist I can revert this line and only add the new "Edit metadata" string, but the option description won't entirely make sense.